letta/llm_api/llm_api_tools.py

```diff
@@ -126,6 +127,7 @@ def create(
         from letta.settings import model_settings
 
         model_settings = model_settings
+        assert isinstance(model_settings, ModelSettings)
```

Comment:
why this assert?

Reply:
used it to fix the type inference in pylance in vscode (to make the highlights better)